lib/tasks/datarepo.rake

```diff
@@ -58,7 +58,7 @@ namespace :datarepo do
       user = User.find_by({email: email})
 
       if user.nil?
-        user = User.new(email: email, uid: email, provider: 'cas')
+        user = User.new(email: email, uid: email.split('@')[0], provider: 'cas')
```

Comment:
Did they change the rule this week or you didn't test it when you submit https://github.com/VTUL/data-repo/pull/154 

Can you write a test case for it?